MANIFEST

```diff
@@ -881,48 +881,34 @@ t/complex/xii.xml
 t/daemon/amsarticle.tex
 t/daemon/broken.tex
 t/daemon/api/port.spec
-t/daemon/api/port.status
```

Comment:
Is there perhaps any git-related reason that we may want to *not* delete these now, but simply ignore them. And just update (possibly) the contents with a later change?

Reply:
I think they pollute the filesystem+manifest quite a bit, and could be replaced by a generic "expect status_code 0 unless explicitly requested otherwise". Then one could change the daemon tests from e.g.
```perl
latexml_tests('t/daemon/fatal');
```

to
```perl
latexml_tests('t/daemon/fatal',
  expect => {
    status_code => 3
  });
```

So fingers crossed the files don't return. The string status is a lot more volatile (e.g. some messages have ids that will need to be maintained, or timestamps or...), and what we really want to test is the _code_ rather than the specific message.

Reply:
ok